### PR TITLE
[OYPD-514] Clean up before decoupled tests.

### PR DESCRIPTION
**Pull request by ivan-berezhnov**
Make sure these boxes are checked before asking for review of your pull request - thank you!

## General checks
- [x] All coding styles are fulfilled and there are no any issues reported by CodeSniffer CI. <br/>CI code sniffer errors
- [x] All tests are running and there are no failed tests reported by CI. <br/>Behat test results
- [x] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [x] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes. <br/>Steps for review
- [x] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [x] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there. <br/>drupal.org email
- [x] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

# Jira issue: 
https://propeople-us.atlassian.net/browse/OYPD-514
Drupal.org issue: https://www.drupal.org/node/2889788

# Steps to review:
- [x] login as admin
- [x] go to [sitename]/admin/modules
- [x] check all OpenY modules in one package 
- [x] check all OpenY modules have same style for name
- [x] go to [sitename]/docs/Development/Upgrade%20path.md#modules-version
- [x] verify section module version removed

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1732 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1732 |
| Installation process | http://install.openy-dev.ffwua.com/build1732/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1732/

**Comment by gundevel** [comment]
Decoupling report http://ci.openymca.org/build1732

**Comment by gundevel** [comment]
Accessibility report http://ci.openymca.org/build1732

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1732/profiles/contrib/openy/build/reports/behat

**Comment by ivan-berezhnov** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1740 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1740 |
| Installation process | http://install.openy-dev.ffwua.com/build1740/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1740/

**Comment by gundevel** [comment]
Accessibility report http://ci.openymca.org/build1740

**Comment by gundevel** [comment]
Decoupling report http://ci.openymca.org/build1740

File: modules/custom/openy_hours_formatter/openy_hours_formatter.info.yml [hunk 1]
```diff
@@ -2,5 +2,6 @@ name: Today\'s Hours
 type: module
```
Comment (alexschedrov) [inline modules/custom/openy_hours_formatter/openy_hours_formatter.info.yml:2]: Can we change name to "OpenY Hours Formatter"?

File: modules/custom/openy_hours_formatter/openy_hours_formatter.info.yml [hunk 2]
```diff
@@ -2,5 +2,6 @@ name: Today\'s Hours
...
 description: Displays today\'s working hours
```
Comment (alexschedrov) [inline modules/custom/openy_hours_formatter/openy_hours_formatter.info.yml]: Let's set "Displays working hours" to remove escaping.

**Review by alexschedrov — CHANGES_REQUESTED** [review]

File: modules/custom/openy_hours_formatter/openy_hours_formatter.info.yml (hunk 1 above)
Comment (ivan-berezhnov) [reply modules/custom/openy_hours_formatter/openy_hours_formatter.info.yml:2]: Yes. Changed.

**Review by alexschedrov — APPROVED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1746 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1746 |
| Installation process | http://install.openy-dev.ffwua.com/build1746/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1746/

**Comment by bjwilkins** [comment]
Decoupling report http://ci.openymca.org/build1746

**Comment by bjwilkins** [comment]
Accessibility report http://ci.openymca.org/build1746

**Review by alexschedrov — APPROVED** [review]

**Pull request merged**